pkgs/development/python-modules/tinytuya/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cryptography
+, requests
+, colorama
+}:
+
+buildPythonPackage rec {
+  pname = "tinytuya";
+  version = "1.13.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "jasonacox";
+    repo = "${pname}";
```

```suggestion
    repo = "tinytuya";
```

I think that there is still no general consents about this but inside the Python eco-system usually we go the route with `repo = "repo-name";`